### ISSUE TITLE
[BUG] node_modules/@whiskeysockets/baileys/lib/Store/make-cache-manager-store.d.ts:2:50 - error TS2552: Cannot find name 'Store'.

### ISSUE THREAD

**Issue by tuin007** [WhiskeySockets/Baileys#608]
Alguém passando por isso ao rodar com TS

node_modules/@whiskeysockets/baileys/lib/Store/make-cache-manager-store.d.ts:2:50 - error TS2552: Cannot find name 'Store'. Did you mean 'Storage'?

2 declare const makeCacheManagerAuthState: (store: Store, sessionKey: string) => Promise<{
                                                   ~~~~~

  node_modules/typescript/lib/lib.dom.d.ts:21652:13
    21652 declare var Storage: {
                      ~~~~~~~
    'Storage' is declared here.

Found 1 error in node_modules/@whiskeysockets/baileys/lib/Store/make-cache-manager-store.d.ts:2

**Comment by barkerpedro** [comment]
Fix in #611

**Comment by barkerpedro** [comment]
> how to fix this?

Fix #611
